README.md

```diff
@@ -526,15 +526,44 @@ enum ImageOrientation {
 
 | Param | Type |
 | --- | --- |
-| environment | Environment | 
-| [venueApiUrl] | string | 
+| environment | Environment |
+| [venueApiUrl] | string |
 
 * getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
+* getSeatAttributesBySeatId(venueId, seatIdCollection) ⇒ Array&lt;SeatAttributes&gt;
+* getDetails(venueId, seatIdCollection) ⇒ Array&lt;VenueDetails&gt;
+* getChartDetails(productId, date) ⇒ Array&lt;ChartDetails&gt;
 
 #### ➥ getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;
 *Get list of seat attributes*
-**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models* 
+**Returns**: Array&lt;SeatAttributes&gt; - *seat attribute models*
+
+| Param | Type |
+| --- | --- |
+| venueId | string |
```

Comment:
Parameters described here are not correct anymore since this function is taken an object

Reply:
done

Reply:
But it's still wrong. Function takes an object, table describes the string

Reply:
@PaulRumyantsevItr , it's provided to api-provider as object. For getSeatAttributes repository it accept simple arguments

Reply:
`const getSeatAttributes = async ({ venueId, seatIdCollection, performanceDate, performanceTime }: Params): Promise<ApiSeatAttributes[]>` - this is what I see in the code

`getSeatAttributes(venueId) ⇒ Array&lt;SeatAttributes&gt;` - this is what I see in the documentation

They should be the same

Reply:
in README.md we descrie Repository methods:
- getVenueServiceRepository = (environment: Environment, venueApiUrl?: string, widgetTitle?: string)
- getSeatAttributesBySeatIds = async (venueId: string, seatIdCollection: string[])
- getDetails = async (venueId: string)
- getChartDetails = async (productId: string, date?: string)

which use getVenueServiceApi methods:
- getSeatAttributes = async ({ venueId, seatIdCollection, performanceDate, performanceTime }: Params): Promise<ApiSeatAttributes[]>
- getDetails = async (venueId: string): Promise<VenueDetails>
- getChartDetails = async (productId: string, date?: string): Promise<ChartDetails>

so, getVenueServiceRepository and getSeatAttributesBySeatIds using getSeatAttributes, but it should't be described i think because we don't describe getVenueServiceApi methods in Readme.